packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneAttachmentBlobs.spec.ts

```diff
@@ -343,8 +343,8 @@ describeCompat("GC attachment blob tombstone tests", "NoCompat", (getTestObjectP
 				},
 			],
 			async () => {
-				// Override ThrowOnTombstoneLoad setting to off.
-				configProvider.set("Fluid.GarbageCollection.ThrowOnTombstoneLoadOverride", false);
+				// Disable ThrowOnTombstoneLoad setting.
```

Comment:
Is Sweep otherwise enabled for these tests?

Reply:
Yeah, but the sweep timeout override is not set. So, the sweep timeout is in days.